modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -0,0 +1,383 @@
+                    />
+                  }
+                  label="Labels"
+                />
+                <FormControlLabel
+                  control={
+                    <Checkbox
+                      checked={identifiersFormat}
+                      onChange={(event) => { setIdentifiersFormat(event.target.checked);}}
+                      name="identifiers"
+                    />
+                  }
+                  label="Identifiers"
+                />
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='baseline' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Data format:</Typography></Grid>
+              <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="values"
+                  aria-label="data"
+                  name="data"
+                  value={hasLabeles}
+                  onChange={(event) => setHasLabeles(Boolean(event.target.value))}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Labels" />
+                  <FormControlLabel value={false} control={<Radio />} label="Values" />
+                </RadioGroup>
+              </Grid>
+            </Grid>
+
+            <Grid container alignItems='baseline' direction="row" className={classes.container}>
+              <Grid item xs={4}><Typography variant="subtitle2">Column selection:</Typography></Grid>
+                <Grid item xs={8}>
+                <RadioGroup
+                  row
+                  defaultValue="isinclude"
+                  aria-label="isinclude"
+                  name="isinclude"
+                  value={isInclude}
+                  onChange={(event) => setIsInclude(Boolean(event.target.value))}
+                >
+                  <FormControlLabel value={true} control={<Radio />} label="Include" />
+                  <FormControlLabel value={false} control={<Radio />} label="Exclude" />
+                </RadioGroup>
+
+                {/* List the entered values */}
+                <div className={classes.entryContainer} >
+                { entities?.filter(v => questionIds.includes(v.path)).map((value, index) =>
+                  <Grid container
+                    key={`${value.name}-${index}`}
+                    direction="row"
+                    justifyContent="space-between"
+                    alignItems="stretch"
+                    className={classes.valueEntry}
+                  >
+                    <Grid item xs={9}>
+                      <Avatar style={{backgroundColor: entitySpecs[value.type].avatarColor || "black"}} className={classes.avatar}>
+                        { entitySpecs[value.type].avatar ? <Icon>{entitySpecs[value.type].avatar}</Icon> : value.type?.charAt(0) }
+                      </Avatar>
+                      <ListItemText primary={value.name} secondary={value.text} />
```

Comment:
We should display the section label when it exists
```suggestion
                      <ListItemText primary={value.name} secondary={value.text || value.label} />
```